utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -70,4 +85,51 @@ class UrlImageParser(
         drawable!!.draw(canvas)
     }
   }
+
+//  TODO below implementation is to load image using Asynctask
+//  override fun getDrawable(urlString: String): Drawable {
+//    IMAGE_DOWNLOAD_URL_TEMPLATE = entity_type + "/" + entity_id + "/assets/image/"
+//
+//    val d = LevelListDrawable()
+//    val empty = context.getResources().getDrawable(R.drawable.ic_menu_gallery)
+//    d.addLevel(0, 0, empty)
+//    d.setBounds(0, 0, empty.getIntrinsicWidth(), empty.getIntrinsicHeight())
+//
+//    LoadImage().execute(GCS_PREFIX + GCS_RESOURCE_BUCKET_NAME + IMAGE_DOWNLOAD_URL_TEMPLATE + urlString, d)
+//    return d
+//  }
+//
+//  internal inner class LoadImage : AsyncTask<Any, Void, Bitmap>() {
+//
+//    private var mDrawable: LevelListDrawable? = null
+//
+//    override fun doInBackground(vararg params: Any): Bitmap? {
+//      val source = params[0] as String
+//      mDrawable = params[1] as LevelListDrawable
+//      Log.d(TAG, "doInBackground $source")
+//      try {
+//        val `is` = URL(source).openStream()
+//        return BitmapFactory.decodeStream(`is`)
+//      } catch (e: FileNotFoundException) {
+//        e.printStackTrace()
+//      } catch (e: MalformedURLException) {
+//        e.printStackTrace()
+//      } catch (e: IOException) {
+//        e.printStackTrace()
+//      }
+//      return null
+//    }
+//
+//    override fun onPostExecute(bitmap: Bitmap?) {
+//      if (bitmap != null) {
+//        val d = BitmapDrawable(bitmap)
+//        mDrawable!!.addLevel(1, 1, d)
+//        mDrawable!!.setBounds(0, 0, bitmap.width, bitmap.height)
+//        mDrawable!!.level = 1
+//        // to refresh TextView
+//        val t = tvContents.getText()
+//        tvContents.setText(t)
+//      }
+//    }
+//  }
```

Comment:
@veena14cs Do we need this commented code? If so, update your TODO correctly to make it clear and also if possible link this to a issue.

Reply:
removed commented code.